File: 백준/Bronze/10813. 공 바꾸기/공 바꾸기.js
const fs = require('fs');
const input = fs.readFileSync(0, 'utf-8').trim().split('\n');

const [N, M] = input[0].split(" ").map(Number);
let basket = Array.from({ length: N }, (_, i) => i + 1);

for (let n = 0; n < M; n++) {
  const [i, j] = input[n + 1].split(" ").map(Number);
  const beforeI = basket[i - 1];
  basket[i - 1] = basket[j - 1];
  basket[j - 1] = beforeI;
}

console.log(basket.join(" "));

File: 백준/Bronze/10813. 공 바꾸기/공 바꾸기.swift
let cond: [Int] = readLine()!.split(separator: " ").map { Int($0)! }
let (n, m): (Int, Int) = (cond[0], cond[1])
var basket: [String] = Array(1 ... n).map { String($0) }

for _ in 0 ..< m {
  let input: [Int] = readLine()!.split(separator: " ").map { Int($0)! }
  basket.swapAt(input[0] - 1, input[1] - 1)
}
print(basket.joined(separator: " "))